features/base/participants/actions.js

```diff
@@ -250,3 +250,24 @@ export function remoteParticipantJoined(id, user = {}) {
         }
     };
 }
+
+/**
+ * Removes local participant.
+ *
+ * @returns {Function}
+ */
+export function removeLocalParticipant() {
+    return (dispatch, getState) => {
+        let localParticipant = getState()['features/base/participants']
+            .find(p => p.local);
+
+        if (localParticipant) {
+            dispatch({
+                type: PARTICIPANT_REMOVED,
```

Comment:
@legastero fixed this in his PR https://github.com/jitsi/jitsi-meet-react/pull/34 (commits https://github.com/jitsi/jitsi-meet-react/pull/34/commits/12a5ef11586f0ecdb8e9b9b1a42d2707c0b3489d and https://github.com/jitsi/jitsi-meet-react/pull/34/commits/8ab3a84247ec10b9f32b8898e69e5c1c7738e00b). So I don't see much sense re-implementing his work here. WDYT?